src/react-components/settings-menu.js

```diff
@@ -430,14 +438,17 @@ export default class SettingsMenu extends Component {
     return (
       <div>
         {!this.props.showAsOverlay && (
-          <FontAwesomeIcon
-            icon={faBars}
+          <div
+            role="button"
```

Comment:
Should this be a `<button>` instead of a `<div>`? (This may affect styling due to default browser styles)

Reply:
Yeah, we definitely ought to use `<button>` or `<a>` tags for clickable elements, instead of `<div>`s. I think I would have had to spend a lot of time fiddling with CSS if I made that change here, and I wanted to avoid that because of the upcoming redesign.